Fakebook.Profile/Fakebook.Profile.UnitTests/APITests/ProfilePictureControllerTests.cs

```diff
@@ -34,10 +42,10 @@ public async Task UploadValidImageExtension()
                 .Returns(Task.FromResult(new Uri("https://www.fake.com")));
             byte[] data = new byte[1000];
             var stream = new MemoryStream(data);
-            var file = new FormFile(stream, 0, 1000, "Data", "dummy.jpeg")
+            var file = new FormFile(stream, 0, 1000, "Data", $"dummy.{extension}")
             {
                 Headers = new HeaderDictionary(),
-                ContentType = "image/jpeg"
+                ContentType = $"image/{extension}"
```

Comment:
- `image/jpg`, `image/svg`, and maybe a couple others don't exist. it's probably better to keep this as `application/octet-stream`, unless you do plan on having the frontend guess the content-type from the extension and send it that way.
- i don't think we need this headerdictionary to be set here.
- you can simplify these tests by using a bool parameter like `successExpected`.

Reply:
* changed to use application/octet-stream
* without the headerdictionary it throws errors
* changed to use a helper method